docs/root/configuration/http/http_filters/ext_authz_filter.rst

```diff
@@ -131,3 +131,12 @@ The HTTP filter outputs statistics in the *cluster.<route target cluster>.ext_au
   denied, Counter, Total responses from the authorizations service that were to deny the traffic.
   failure_mode_allowed, Counter, "Total requests that were error(s) but were allowed through because
   of failure_mode_allow set to true."
+
+Runtime
+-------
+The fraction of requests for which the filter is enabled can be configured via the the
+:ref:`runtime_key <envoy_api_msg_core.runtimefractionalpercent>` value of the :ref:`filter_enabled
+<envoy_api_msg_config.filter.http.ext_authz.v2.ExtAuthz>` field.
+
+To determine if the filter is enabled you can check the runtime values via the admin panel at
```

Comment:
nit: I would probably kill this as we don't do this elsewhere, but not a big deal.